gui/default/syncthing/folder/folderPickerModalView.html

```diff
@@ -0,0 +1,30 @@
+<modal id="folderPicker" status="default" heading="{{'Select Folder' | translate}}" icon="fa fa-folder-open" large="yes"
+       closeable="yes">
+  <div class="modal-body" ng-controller="FolderPickerModalController">
+    <div class="form-group">
+      <label for="folderPickerPath"><span translate>Folder Path</span></label>
+      <input id="folderPickerPath" class="form-control" ng-change="selectNodeByCurrentPath()" ng-model="currentPath" type="text">
+    </div>
+    <hr />
+    <div id="folderTree-container" class="tree-container">
+      <table id="folderTree" class="tree">
+        <tbody>
+        <tr>
+          <td>
+            <span translate>Loading data...</span>
+          </td>
+        </tr>
+        </tbody>
```

Comment:
Please fix the indentation (`<tbody>` and `<tr>` are on the same level right now).